src/components/menus/base-equipment-menu.tsx

```diff
@@ -244,19 +295,19 @@ const BaseEquipmentMenu = ({
                     >
                         {/* menus for the substation */}
                         <ViewInSpreadsheetItem
-                            key={equipment.substationId}
-                            equipmentType={EQUIPMENT_TYPES.SUBSTATION}
-                            equipmentId={equipment.substationId}
+                            key={equipment.id}
+                            equipmentType={EquipmentType.SUBSTATION}
+                            equipmentId={equipment.id}
                             itemText={getNameOrId({
-                                name: equipment.substationName,
-                                id: equipment.substationId,
+                                name: equipment.id,
+                                id: equipment.id,
```

Comment:
Because now, for maps, the equipment is of type GSMapEquipments, which inherits from Equipment.
For SLDs, we create the type equipment={{ id: equipmentMenu.equipmentId } as Equipment} https://github.com/gridsuite/gridstudy-app/pull/2339/files#diff-0a85d6348525cd3105a6ad6c040ce5c22b8890b5addfb26f62dd8abf437edb69R382